Add origami-version GitHub Action

JakeChampion opened the pull request.

origamiserviceuser commented:
o-normalise bundle size difference from 2.0.1 to 2b7606970aa28efeb884cb3f803de00b23179a7b
No significant bundle size differences found.

JakeChampion committed:
Add origami-version GitHub Action

When this is merged, we can have automated releases by using the release-type labels "major"/"minor"/"patch" on a pull-request before merging the pull-request.

origamiserviceuser commented:
o-normalise bundle size difference from 2.0.1 to f8723fa1992291f901fc840a97bd3575b5c6b64f
No significant bundle size differences found.

JakeChampion commented:
This helps achieve the OKR outlined here https://github.com/Financial-Times/origami/issues/39
